### PR TITLE
Features/35804 drawer modifcation

### PR DESCRIPTION
**Pull request by dheeraja00**
This contain drawer & side navigation modifications

### CODE REVIEW

File: src/components/Drawer/DrawerContent.tsx [hunk 1]
```diff
@@ -55,31 +55,29 @@ class DrawerContent extends React.Component<Props, never> {
   }
 
   render() {
-    const containerClassName = this.getContainerClassName();
-    const barClassName = this.getBarClassName();
+    // const containerClassName = this.getContainerClassName();
+    // const barClassName = this.getBarClassName();
```
Comment (ankitsaraogi) [inline src/components/Drawer/DrawerContent.tsx]: remove commented code

File: src/components/SideNavigation/SideNavigation.tsx [hunk 2]
```diff
@@ -1,11 +1,15 @@
 
 import * as React from 'react';
 import { themr, ThemedComponentClass } from 'react-css-themr';
+import { classNames } from '@shopify/react-utilities/styles';
+
 import Icon from '../Icon';
-import { SIDENAVIGATION } from '../ThemeIdentifiers';
+import { FlexBox } from '..';
```
Comment (ankitsaraogi) [inline src/components/SideNavigation/SideNavigation.tsx]: This looks wrong.

**Comment by dheeraja00** [comment]
@ankitsaraogi taken care of the comments

**Comment by ankitsaraogi** [comment]
@dheeraja00 : Can you just check if build is failing or is it all fine.

**Comment by dheeraja00** [comment]
@ankitsaraogi this is solved

**Pull request merged**